### PR TITLE
update native sdks to 12.0.0

### PR DESCRIPTION
**Pull request by jamesdixon**

### CODE REVIEW

File: intercom_flutter/pubspec.yaml [hunk 1]
```diff
@@ -29,5 +29,5 @@ flutter:
         default_package: intercom_flutter_web
 
 environment:
-  sdk: '>=2.12.0 <3.0.0'
+  sdk: ">=2.12.0 <3.0.0"
```
Comment (Zazo032) [inline intercom_flutter/pubspec.yaml:32]: Why is this changed?

**Review by Zazo032 — APPROVED** [review]
LGTM if there are no breaking changes on the SDK side

File: intercom_flutter/pubspec.yaml (hunk 1 above)
Comment (jamesdixon) [reply intercom_flutter/pubspec.yaml:32]: It was automatically corrected by Dart DevTools. It looks like they prefer double quotes. Example from the Flutter site: https://docs.flutter.dev/development/tools/pubspec

**Review by deepak786 — APPROVED** [review]
LGTM.

have you run the example app after these changes?

**Comment by jamesdixon** [comment]
@deepak786 yes, I did and the messenger displays as expected.
